packages/api/test/perf/compileRouteUrlFormater.test.ts

```diff
@@ -3,6 +3,7 @@ import {compileRouteUrlFormater} from "../../src/utils/urlFormat.js";
 /* eslint-disable no-console */
 
 describe("route parse", () => {
+  // eslint-disable-next-line vitest/no-disabled-tests
```

Comment:
I dont think we should remove all the skipped test warning under this PR.  Feels like these should be addressed under separate PR cover so that it is clear why they all went away and so it easier to find when they get removed or turned back on